### PR TITLE
Added question "But what universe is this?"

### PR DESCRIPTION
**Pull request by jhonnyjblaze**
"Please review this, @casey-collab"

### CODE REVIEW

File: english/blaze-verse/blaze_verse.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+wandering through the vastness of this universe.
```
Comment (casey-collab) [inline english/blaze-verse/blaze_verse.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jhonnyjblaze** [comment]
Hi @casey-collab is that what you mean or am I still lost?

**Comment by casey-collab** [comment]
Unfortunately, pull requests adding more than 100 lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to add 100 lines or fewer.

**Pull request closed without merging**